definitions/npm/firebase_v4.x.x/flow_v0.46.x-/firebase_v4.x.x.js

```diff
@@ -0,0 +1,410 @@
+/* @flow */
+
+/**** firebase ****/
+
+declare interface FirebaseConfig {
+  apiKey: string;
+  authDomain?: string;
+  databaseURL: string;
+  projectId: string;
```

Comment:
some of the example code in firebase does show projectid and I believe that is only recently introduced. anyway since firebase does not treat it as mandatory, we should not treat it as mandatory too... feel free to send a PR to make projectId optional